src/main/java/com/kuljava/swiatwsi/rawmaterials/Scheduler.java

```diff
@@ -0,0 +1,30 @@
+package com.kuljava.swiatwsi.rawmaterials;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Scheduler {
+
+  @Autowired RawMaterialsRepository repository;
+
+  private Long woodIncrease;
+  private Long clayIncrease;
+  private Long ironIncrease;
+
+  public Scheduler(
+      @Value("${rawmaterials.scheduler.woodincrease}") String woodIncrease,
```

Comment:
If this is taken from parameters, You don't have to inject it in the constructor. Just @ value annotation above the fields would be ideal.

Reply:
Yes, but actually no. @ Value annotation gives a string so I need to convert it to Long

Reply:
Sooo maybe @ ConfigurationProperties class? https://www.baeldung.com/configuration-properties-in-spring-boot